haystack/modeling/model/language_model.py

```diff
@@ -1570,3 +1574,101 @@ def enable_hidden_states_output(self):
 
     def disable_hidden_states_output(self):
         self.model.encoder.config.output_hidden_states = False
+
+
+class DebertaV2(LanguageModel):
+    """
+    This is a wrapper around the DebertaV2 model from HuggingFace's transformers library.
+    It is also compatible with DebertaV3 as DebertaV3 only changes the pretraining procedure.
+
+    NOTE:
+    - DebertaV2 does not output the pooled_output. An additional pooler is initialized.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.model = None
+        self.name = "deberta-v2"
+        self.pooler = None
+
+    @classmethod
+    @silence_transformers_logs
+    def load(cls, pretrained_model_name_or_path: Union[Path, str], language: str = None, **kwargs):
+        """
+        Load a pretrained model by supplying
+
+        * the name of a remote model on s3 ("google/electra-base-discriminator" ...)
```

Comment:
maybe we can give an DeBERTa model as an example here?

Reply:
`microsoft/deberta-v3-base` and `microsoft/deberta-v3-large` for example. Also "the name of a remote model on s3" should better be something like "a remote name from Huggingface's modelhub or a local path." as this usage is much more common.